libhb/encavcodec.c

```diff
@@ -101,6 +126,18 @@ int encavcodecInit( hb_work_object_t * w, hb_job_t * job )
         {
             hb_log("encavcodecInit: VP9 encoder");
         } break;
+        case AV_CODEC_ID_H264:
+        {
+            codec = avcodec_find_encoder_by_name("h264_nvenc");
```

Comment:
Although not strictly necessary at this time, this should be:
```
case AV_CODEC_ID_H264:
{
    if (job->vcodec == HB_VCODEC_NVENC_H264)
    {
        codec = avcodec_find_encoder_by_name("h264_nvenc");
    }
    else
    {
        hb_error("encavcodecInit: unsupported encoder!");
        ret = 1;
        goto done;
    }
} break;
```
In the future, there will be other h.264 encoders that use this path.